library/cwm/src/lib/cwm/common_widgets.rb

```diff
@@ -365,6 +365,40 @@ class RichText < AbstractWidget
     self.widget_type = :richtext
 
     include ValueBasedWidget
+
+    # Determines if the scroll should be restored after updating the content
+    #
+    # @return [Boolean] true if the scroll should be restored; false otherwise
+    def keep_scroll?
+      false
```

Comment:
Um, maybe the default value should be `true`. I think the more common case is to keep the scroll position after changing the RichText content.

What do you think?

Reply:
I agree, the most common case is (or should be, IMO) to keep the scroll position. However, I used `false` as a default value because I'm not sure if changing the behavior will be break things.

So, probably we should collect more opinions before taking a final decision, shouldn't we?
Probably we should collect a few more opinions, to tak

Reply:
As discussed in the #yast channel at the freenode.net IRC server, we are to keep returning `false` as default because updating to a different content will produce not expected behavior. This _feature_ is intended to be used with the same or almost the same content, to give a sense of continuity when the widget is _redrawn_.